DoctrineBundle.php

```diff
@@ -46,6 +46,10 @@ public function build(ContainerBuilder $container)
         $container->addCompilerPass(new class () implements CompilerPassInterface {
             public function process(ContainerBuilder $container): void
             {
+                if (! $container->findTaggedServiceIds('lock.store')) {
```

Comment:
I would rather not have this, the only call in that subscriber outside foreach is rather cheap.

Reply:
So if it doesn't cause an error, as it does for session.handler service, we should rather let it be loaded knowing that the component is not installed ? 

Thanks for you feedback !

Reply:
Part of the overhead in Symfony is listeners registered that don't do much so I'd prefer core bundles like this one do their cleanups. But I'm fine discussing this in a dedicated PR for all listeners at once, so let's remove this from the current PR.

Reply:
ok !